dbt/adapters/clickhouse/impl.py

```diff
@@ -233,6 +233,23 @@ def get_csv_data(self, table):
 
         return buf.getvalue()
 
+    def run_sql_for_tests(self, sql, fetch, conn):
+        cursor = conn.handle
+        try:
+            result = cursor.execute(sql)
+            if fetch == "one" and len(result) > 0:
+                return result[0]
+            elif fetch == "all":
+                return result
+            else:
+                return
+        except BaseException as e:
+            print(sql)
```

Comment:
Maybe we could use std logger. What do you think?

Reply:
Good idea, I used AdapterLogger from dbt